tests/model_execution/test_bash.py

```diff
@@ -882,6 +884,7 @@ def test_analysis_settings_4_0_reins_iters_chunk(self):
 class Genbash_GulItemStream(Genbash):
     @classmethod
     def setUpClass(cls):
+        super().setUpClass()
```

Comment:
Without calling `super()` we cannot run individual tests or test classes (with pytest or unittest) and we are forced to run the full file.

```
$ pytest ./tests/model_execution/test_bash.py::Genbash_TempDir::test_il_summarycalc_20_partition_chunk
...
        params = bash_params(
            max_process_id=num_partitions,
            analysis_settings=analysis_settings,
            num_reinsurance_iterations=(num_reinsurance_iterations or self.ri_iterations),
            fifo_tmp_dir=(fifo_tmp_dir or self.fifo_tmp_dir),
            stderr_guard=(stderr_guard or self.stderr_guard),
            gul_alloc_rule=(gul_alloc_rule or self.gul_alloc_rule),
            il_alloc_rule=(il_alloc_rule or self.il_alloc_rule),
            ri_alloc_rule=(ri_alloc_rule or self.ri_alloc_rule),
>           num_gul_per_lb=self.num_gul_per_lb,
            num_fm_per_lb=self.num_fm_per_lb,
            event_shuffle=self.event_shuffle,
            bash_trace=(bash_trace or self.bash_trace),
            gul_legacy_stream=(gul_legacy_stream or self.gul_legacy_stream),
            fmpy=(fmpy or self.fmpy),
        )
E       AttributeError: 'Genbash_TempDir' object has no attribute 'num_gul_per_lb'

tests/model_execution/test_bash.py:132: AttributeError
================================================================================= short test summary info =================================================================================
FAILED tests/model_execution/test_bash.py::Genbash_TempDir::test_il_summarycalc_20_partition_chunk - AttributeError: 'Genbash_TempDir' object has no attribute 'num_gul_per_lb'
============================================================================== 1 failed, 2 warnings in 0.47s ==============================================================================
```